Src/AutoMoqUnitTest/MockVirtualMethodsCommandTest.cs

```diff
@@ -129,13 +129,13 @@ public void SetsUpPropertyGetters_ToRetrieveReturnValueFromContext()
             // Fixture setup
             var fixture = new Fixture();
             var frozenString = fixture.Freeze<string>();
-            var mock = new Mock<IInterfaceWithProperty>();
+            var mock = new Mock<IInterfaceWithGetOnlyProperty>();
```

Comment:
Instead of changing the existing test to verify new behavior, could you please:
1. Change this test to assert that get-set properties are ignored.
2. Add a test that get-set properties are ignored for classes.
3. Add a standalone test that get-only properties are set up for both interfaces and classes.

P.S. Pay attention to the types from the `TestTypeFoundation` project - you could potentially reuse some of them 😉